proto/eth/v2/sync_committee.pb.go

```diff
@@ -0,0 +1,340 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.31.0
+// 	protoc        v4.25.1
+// source: proto/eth/v2/sync_committee.proto
+
```

Comment:
sync_committee.pb.go file should be removed. run `bazel clean` and rerun scripts.